### PR TITLE
HUBOT_JIRA_GITHUB_DISABLED

### PR DESCRIPTION
**Pull request by ndaversa**

### CODE REVIEW

File: src/index.coffee [hunk 1]
```diff
@@ -102,11 +106,11 @@ class JiraBot
         _attachments.push ticket.toAttachment()
         ticket
       .then (ticket) ->
-        Github.PullRequests.fromKey ticket.key
+        Github.PullRequests.fromKey ticket.key unless Config.github.disabled
       .then (prs) ->
-        prs.toAttachment()
+        prs?.toAttachment()
```
Comment (sylvesterjakubowski) [inline src/index.coffee]: much cleaner, thanks @ndaversa

**Comment by sylvesterjakubowski** [comment]
@ndaversa the documentation doesn't seem updated on NPM, I thought I changed the correct file but it's not there. 

https://www.npmjs.com/package/hubot-jira-bot

**Comment by ndaversa** [comment]
@sylvesterjakubowski yea that's odd. I believe the right changes have been made. It might be a cache issue with npm. I did publish a new version with these changes though.

**Comment by sylvesterjakubowski** [comment]
Not sure because it does show the new version you pushed @ndaversa , but no docs.

Oh well, not sure if anyone else will be able to take advantage but it was a big deal for us.

**Pull request merged**